assets/javascripts/discourse/templates/modal/login-prompt.hbs

```diff
@@ -0,0 +1,31 @@
+{{#d-modal-body title="expose_emails_in.login_prompt.title" class="login-prompt"}}
+  <div>
+    <h3>
+      {{#if model.stranger_danger}}
+        {{i18n "expose_emails_in.login_prompt.body_stranger"}}
+      {{else}}
+        {{i18n "expose_emails_in.login_prompt.body"}}
+      {{/if}}
+      {{d-button  class="btn-primary login-button"
```

Comment:
Yeah, it's just to make all the attributes line up (the extra space is to make it tab-aligned) - not sure if it's a standard, but I think I've seen it used elsewhere